mrp_lot_cost/__openerp__.py

```diff
@@ -0,0 +1,35 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    @authors: Alexander Ezquevo <alexander@acysos.com>
+#    Copyright (C) 2015  Acysos S.L.
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    "name": "MRP Lot Unit Cost",
+    "version": "1",
```

Comment:
Please follow the guidelines from (8.0.1.0.0):
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md